speech/src/main/java/com/example/speech/InfiniteStreamRecognize.java

```diff
@@ -39,19 +40,16 @@
 import javax.sound.sampled.AudioSystem;
 import javax.sound.sampled.DataLine;
 import javax.sound.sampled.DataLine.Info;
+import javax.sound.sampled.LineUnavailableException;
 import javax.sound.sampled.TargetDataLine;
 
 public class InfiniteStreamRecognize {
 
   private static final int STREAMING_LIMIT = 290000; // ~5 minutes
```

Comment:
nit: This code sample is called _infinite_ streaming. The name "streaming limit" can be potentially confusing. Consider providing more meaningful name.